src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/iOS/NativeLabel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using UIKit;
+using RectangleF = CoreGraphics.CGRect;
+using SizeF = CoreGraphics.CGSize;
+
+namespace Xamarin.Platform
+{
+	public class NativeLabel : UILabel
+	{
+		public NativeLabel(RectangleF frame) : base(frame)
+		{
+
+		}
+
+		public UIEdgeInsets TextInsets { get; set; }
+
+		public override void DrawText(RectangleF rect) => base.DrawText(TextInsets.InsetRect(rect));
+
+		public override SizeF SizeThatFits(SizeF size) => AddInsets(base.SizeThatFits(size));
```

Comment:
Can this code be moved to an override on the Handlers? 

If we have this sizing in both places aren't they going to be at odds with each other and the final measure on the platform isn't going to be represented correctly based on what the xplat knows

```C#
	public partial class LabelHandler : AbstractViewHandler<ILabel, UILabel>
	{
		protected override UILabel CreateView() => new UILabel();

		public override SizeRequest GetDesiredSize(double widthConstraint, double heightConstraint)
		{
			return base.GetDesiredSize(widthConstraint, heightConstraint);
		}
	}
```